addons/sale_pdf_quote_builder/models/sale_order.py

```diff
@@ -48,6 +48,20 @@ def _compute_is_pdf_quote_builder_available(self):
                 or order.order_line.available_product_document_ids
             )
 
+    # === ONCHANGE METHODS === #
+
+    @api.onchange('sale_order_template_id')
+    def _onchange_sale_order_template_id(self):
+        super()._onchange_sale_order_template_id()
+        for order in self:
+            available_quotation_document = self.env['quotation.document'].search(AND([
+                self.env['quotation.document']._check_company_domain(order.company_id),
+                [('quotation_template_ids', 'in', [False, order.sale_order_template_id.id])]
+            ]))
+
+            # Remove documents which are no longer available.
+            self.quotation_document_ids &= available_quotation_document
```

Comment:
```suggestion
            # Remove documents which are no longer available.
            self.quotation_document_ids &= available_quotation_documents
```
we can remove the empty line, and the variable name would be better pluralized.